src/main/java/appeng/recipes/game/WirelessTerminalRecipe.java

```diff
@@ -0,0 +1,142 @@
+package appeng.recipes.game;
+
+import appeng.api.AEApi;
+import appeng.api.definitions.IComparableDefinition;
+import appeng.api.definitions.IDefinitions;
+import appeng.items.parts.ItemFacade;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'appeng.items.parts.ItemFacade'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AUselessImportCheck)